src/types/BapiProduct.ts

```diff
@@ -103,6 +103,11 @@ export interface Variant {
   attributes?: Attributes;
   advancedAttributes?: AdvancedAttributes;
   lowestPriorPrice?: LowestPriorPrice;
+  searchCategoryIds?: SearchCategoryIds;
+}
+
+export interface SearchCategoryIds {
+  searchCategoryIds: number[];
```

Comment:
I'm pretty sure this is not the correct type since `searchCategoryIds` is a list on the top level: https://api-cloud.aboutyou.de/v1/products?with=searchCategoryIds&page=2000